lib/posix-process/process.h

```diff
@@ -33,13 +33,11 @@
 #ifndef __PROCESS_H_INTERNAL__
```

Comment:
Btw I notice that the headers have __PROCESS_* guards instead of __POSIX_PROCESS_* guards. Is that ok?

Reply:
Yeah, I chose that based on `lib/posix-process/include/uk/process.h`.